src/cursor/cursor.ts

```diff
@@ -54,8 +108,43 @@ export interface CursorOptions extends CoreCursorOptions {
   numberOfRetries?: number;
 }
 
-const kCursor = Symbol('cursor');
+/** @public */
+export interface CursorCloseOptions {
+  /** Bypass calling killCursors when closing the cursor. */
+  skipKillCursors?: boolean;
+}
+
+/** @internal */
+interface InternalCursorState extends BSONSerializeOptions {
```

Comment:
looks like we lost my comment on this, it looks like `InternalCursorState` is no longer used